tests/sort_test.go

```diff
@@ -0,0 +1,15 @@
+package tests
+
+import (
+	"sort"
+	"testing"
+)
+
+func TestSortSlice(t *testing.T) {
+	a := [...]int{5, 4, 3, 2, 1}
+	s := a[1:4]
+	sort.Slice(s, func(i, j int) bool { return s[i] < s[j] })
+	if a != [...]int{5, 2, 3, 4, 1} {
+		t.Fatalf("not equal")
+	}
```

Comment:
How about adding more test cases when sorting `a[:]`, `a[0:0]` and specifying cap (e.g. `a[1:4:5]`)?